.github/workflows/docker-build.yaml

```
@@ -669,12 +654,42 @@ jobs:
+            -   name: GIT checkout branch - ${{ github.head_ref }}
+                uses: actions/checkout@v4
+                with:
+                    ref: ${{ github.head_ref }}
+            -   name: Download all snapshot artifacts
+                uses: actions/download-artifact@v4
+                with:
+                    path: ./snapshots
+                    pattern: cypress-snapshots-*
+                    merge-multiple: true
+            -   name: Commit and push all snapshots
+                run: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)